app/models/user.rb

```diff
@@ -34,5 +39,8 @@ class User < ApplicationRecord
 
   has_many :surveys
 
-  enum use_type: [:organisational, :educational]
+  enum use_type:           [:organisational, :educational]
+  enum org_type:           [:school, :university, :consultancy, :government, :intergovernmental,
```

Comment:
Could any of this change in the future? If we would ever add another org_type or use_type, enum might not be the best option here, read: https://medium.com/@elcortez/the-many-issues-with-rails-enum-36cfb2bec620

Reply:
I based this work off of the `enum use_type` which I already found in this project. However, if you think it would be better to switch to using strings instead? I am happy to make the switch.